community/test-kits.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Test Kits
+nav_order: 2
+layout: docs
+section: community
+---
+
+# Test Kits
+
+Here is the list of Test Kits that the Inferno Team is currently aware of, along with a link to the source code and where it is publicly hosted, if available.
```

Comment:
Could this language be cleaned up a little, feels very informal.  Also, could you add in a link here, which shows inferno test kits that are publicly available on github: https://github.com/inferno-framework/inferno-core/network/dependents

Reply:
can do! Are there other test kits at that link that should be included in the table?